src/bin/cargo-miri.rs

```diff
@@ -109,6 +110,8 @@ fn main() {
                     }
                 }
                 (true, "lib") => {
+                    // For libraries we call `cargo rustc -- --test <rustc args>`
+                    // Notice now that `--test` is a rustc arg rather than a cargo arg.
```

Comment:
"libraries" here means "`#[test]` functions", right?

Reply:
Yes.

Reply:
Okay. Could you add that?

Took me a while to figure that out (originally, as in some months ago, I thought these are also just test binaries for cargo, I didn't know where the magic lived).